textfsm.py

```diff
@@ -981,11 +982,11 @@ def _Operations(self, rule):
     # Lastly process line operators.
     if rule.line_op == 'Error':
       if rule.new_state:
-        raise TextFSMError('Error: %s. Line: %s.'
-                           % (rule.new_state, rule.line_num))
+        raise TextFSMError('Error: %s. Rule_Line: %s. Raw_Line: %s.'
```

Comment:
Thank you jmcgill298 for submitting this change.

Looks good.

Do you need to link the words together, i.e. are you using the '_' to aid in post processing?
Otherwise: 'Error: %s. Rule Line: %s. Input Line: %s.' feels a little more user friendly to me.

Reply:
I'm not doing anything with post processing that depends on the -, so that is fine with me; I have pushed the updates